### PR TITLE
admin/testing-ci

### LINKED ISSUE #87: Add test to pull request workflow
### Feature Description

Add testing to `.github/workflows/test-and-lint.yml`. See [cdp-backend repo](https://github.com/CouncilDataProject/cdp-backend) for relevant example files - `tox.ini`, `setup.cfg`, `setup.py`, etc.

Note, we have `cdp_scrapers/tests/scraper_tests.py`.

### Use Case

Will test pull requests.

### Solution

Run tests after the `Check with Black` step in `test-and-lint.yml`.

### Alternatives

Creating a new GitHub workflow file for testing.

### PR DESCRIPTION
**Pull request by Shak2000**
### Link to Relevant Issue

This pull request resolves #87 

### Description of Changes

I am adding a workflow to run tests after every push. This is the first attempt.

### CODE REVIEW

**Comment by JacksonMaxfield** [comment]
Thanks for getting to this!! Looking good so far just minor changes.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+files: cdp_backend
```
[the remark this answers is not in the document]
Comment (Shak2000) [reply .pre-commit-config.yaml]: Done!

File: tox.ini [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
+    CI
+setenv =
+    PYTHONPATH = {toxinidir}
+extras =
+    test
 commands =
-    flake8 cdp_scrapers --count --verbose --show-source --statistics
-    black --check cdp_scrapers
+    pytest --basetemp={envtmpdir} --cov-report xml --cov-report html --cov=cdp_scrapers cdp_scrapers/tests/scraper_util_tests.py {posargs}
+    pytest --basetemp={envtmpdir} --cov-report xml --cov-report html --cov=cdp_scrapers cdp_scrapers/tests/scraper_tests.py {posargs}
```
[the remark this answers is not in the document]
Comment (Shak2000) [reply tox.ini]: Done!

**Comment by Shak2000** [comment]
@JacksonMaxfield Ready for another review.

**Review by JacksonMaxfield — APPROVED** [review]
Looks good to me! Thanks @Shak2000

**Pull request merged**